docs/security/shiroauthentication.md

```diff
@@ -135,15 +135,18 @@ ldapRealm.contextFactory.environment[ldap.searchBase] = dc=COMPANY,dc=COM
 ldapRealm.contextFactory.url = ldap://ldap.test.com:389
 ldapRealm.userDnTemplate = uid={0},ou=Users,dc=COMPANY,dc=COM
 ldapRealm.contextFactory.authenticationMechanism = SIMPLE
+ldapRealm.contextFactory.systemUsername = cn=admin,dc=COMPANY,dc=COM
+ldapRealm.contextFactory.systemPassword = bindPassword
+securityManager.realms = $ldapRealm
```

Comment:
normally that description should come from ldap server config/admin. the reason is that ldap servers by default (especially production level) don't allow anonymous bind and ldap server needs to know the person who's requesting for ldap command (e.g. ldapsearch)